user-interface-document/index.js

```diff
@@ -3,9 +3,6 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
-// CKEditor Commercial Features require a license key to work properly.
-// * You can get a trial license key: https://orders.ckeditor.com/trial/premium-features.
-// * Or you can comment out (disable) the plugins imported from the "ckeditor5-premium-features" package.
```

Comment:
Other demos have this comment and I think it's worth keeping.

Reply:
I must've removed this while replacing with license keys for tests :facepalm: